views/index.jsx

```diff
@@ -44,13 +45,17 @@ function render({ slug, meta, html }) {
 			<PageBreadcrumb slug={slug} pageTitle={meta.h1} />
 			<ContainerWhenSubpage slug={slug}>
 				{safe(html)}
+
+				<Footer />
 			</ContainerWhenSubpage>
```

Comment:
I take full responsibility for the container catastrophy that exists in our CSS. It is horribly unintuitive and I was embarrassed to commit it, but I wanted to finish 🙈

The `<ContainerWhenSubpage>` will only add the container class when it is _not_ the index page, which is why this doesn't work with the main layout. (as I said... embarrassing).

The reason is that we need multiple containers for the main page so I added them directly in the Markdown files (again... embarrassing), so we do not add a container as a wrapper here.

For a quick fix, I would move the `<Footer />` underneath the `<ContainerWhenSubpage>` and use the `%containerify` placeholder selector on it so that it centers itself as if it were in a container. I'll make a concrete suggestion there on the new CSS.